raiden/tests/utils/mock_client.py

```diff
@@ -268,7 +272,7 @@ def manager_addresses(self):
             for manager in self.token_manager.values()
         ]
 
-    def tokenadded_filter(self):
+    def tokenadded_filter(self, from_block=None, to_block=None):
```

Comment:
`from_block`, `to_block` are not used here. how about just allowing for `**kwargs` instead, so no one expects this to do anything?